sal_interfaces/HVAC/HVAC_Events.xml

```diff
@@ -1,23 +1,92 @@
-    <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_logevent_detailedState</EFDB_Topic>
-    <Description>Event for the detailedState.</Description>
-    <item>
-      <EFDB_Name>detailedState</EFDB_Name>
-      <Description>Detailed state</Description>
-      <IDL_Type>long</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-  </SALEvent>
+  <Enumeration>
+    SummaryState_DisabledState,
+    SummaryState_EnabledState,
+    SummaryState_FaultState,
+    SummaryState_OfflineState,
+    SummaryState_StandbyState
+  </Enumeration>
+  <Enumeration>
+    DisabledState_DisabledState,
+    DisabledState_EnabledState,
+    DisabledState_FaultState,
+    DisabledState_OfflineState,
+    DisabledState_StandbyState
+  </Enumeration>
+  <Enumeration>
+    HvacTopicId_chiller01P01=0X0001,
+    HvacTopicId_chiller02P01=0X0002,
+    HvacTopicId_chiller03P01=0X0003,
+    HvacTopicId_crack01P02=0X0004,
+    HvacTopicId_crack02P02=0X0005,
+    HvacTopicId_fancoil01P02=0X0006,
+    HvacTopicId_fancoil02P02=0X0007,
+    HvacTopicId_fancoil03P02=0X0008,
+    HvacTopicId_fancoil04P02=0X0009,
+    HvacTopicId_fancoil05P02=0X000A,
+    HvacTopicId_fancoil06P02=0X000B,
+    HvacTopicId_fancoil07P02=0X000C,
+    HvacTopicId_fancoil08P02=0X000D,
+    HvacTopicId_fancoil09P02=0X000E,
+    HvacTopicId_fancoil10P02=0X000F,
+    HvacTopicId_fancoil11P02=0X0010,
+    HvacTopicId_fancoil12P02=0X0011,
+    HvacTopicId_manejadoraLower01P05=0X0012,
+    HvacTopicId_manejadoraLower02P05=0X0013,
+    HvacTopicId_manejadoraLower03P05=0X0014,
+    HvacTopicId_manejadoraLower04P05=0X0015,
+    HvacTopicId_manejadoraSblancaP04=0X0016,
+    HvacTopicId_manejadoraSlimpiaP04=0X0017,
+    HvacTopicId_vea01P01=0X0018,
+    HvacTopicId_vea01P05=0X0019,
+    HvacTopicId_vea08P05=0X001A,
+    HvacTopicId_vea09P05=0X001B,
+    HvacTopicId_vea10P05=0X001C,
+    HvacTopicId_vea11P05=0X001D,
+    HvacTopicId_vea12P05=0X001E,
+    HvacTopicId_vea13P05=0X001F,
+    HvacTopicId_vea14P05=0X0020,
+    HvacTopicId_vea15P05=0X0021,
+    HvacTopicId_vea16P05=0X0022,
+    HvacTopicId_vea17P05=0X0023,
+    HvacTopicId_vec01P01=0X0024,
+    HvacTopicId_vin01P01=0X0025,
+    HvacTopicId_vex03LowerP04=0X0026,
+    HvacTopicId_vex04CargaP04=0X0027
+  </Enumeration>
+  <Enumeration>
+    ChillerId_c01=1,
+    ChillerId_c02=2,
+    ChillerId_c03=3
+  </Enumeration>
+  <Enumeration>
+    CrackId_c01=1,
+    CrackId_c02=2
+  </Enumeration>
+  <Enumeration>
+    FancoilId_f01=1,
+    FancoilId_f02=2,
+    FancoilId_f03=3,
+    FancoilId_f04=4,
+    FancoilId_f05=5,
+    FancoilId_f06=6,
+    FancoilId_f07=7,
+    FancoilId_f08=8,
+    FancoilId_f09=9,
+    FancoilId_f10=10,
+    FancoilId_f11=11,
+    FancoilId_f12=12
+  </Enumeration>
+  <Enumeration>
+    ManejadoraId_mSblanca=1,
+    ManejadoraId_mSlimpia=2
+  </Enumeration>
+  <Enumeration>
+    ManejadoraLowerId_m01=1,
+    ManejadoraLowerId_m02=2,
+    ManejadoraLowerId_m03=3,
+    ManejadoraLowerId_m04=4
+  </Enumeration>
```

Comment:
I *strongly* suggest you just use the HvacTopicId enum for configuration commands.

This gives one standard way to refer to a device. It also addresses the issue I raised last time: that your configuration commands aren't safe because a user can use the wrong enum.

A very nice way to handle this would be to assign a range of values for each category of device (Tekniker does this). For example:
```
    DeviceId_Chiller01P01=101,
    DeviceId_Chiller02=102,
    DeviceId_chiller03=103,
    DeviceId_Crack01P02=201,
    DeviceId_Crack02P02=202,
    DeviceId_Fancoil01P02=301,
    ...
    DeviceId_Fancoil12P02=312,
    DeviceId_ManejadoraLower01P05=401
    ...  
```

Then if you need to send indices 1-N to the lower level device, you can subtract the appropriate multiple of 100 -- information you can easily store in another enum, or a Python dict. Of course multiples of 100 aren't necessary at all, but something like that does allow adding and subtracting devices without having to change other values, and it is quite readable.

Reply:
I gently suggest you use base 10 for this enum and reserve hex values for bit masks. Using hex for bit masks is is a common convention, probably because that make it obvious when a large number is a power of 2. For other sequences, base 10 is fine and arguably more readable.

Reply:
A good suggestion and thanks for the short explanation on Slack. Done.